.changelog/27053.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+disconnect: fix disconnect.replace respecting the reschedule block
```

Comment:
This only tells a user that we've changed the behavior, but can we clarify what behavior has changed? (The PR description is actually a little ambiguous as well.)